packages/main/test/samples/Toast.sample.html

```diff
@@ -0,0 +1,215 @@
+
+		<!--TopCenter-->
+		<ui5-button id="wcBtnShowToastTC" class="samples-margin">TopCenter</ui5-button>
+		<ui5-toast id="wcToastTC" placement="TopCenter">TopCenter</ui5-toast>
+
+		<!--TopEnd-->
+		<ui5-button id="wcBtnShowToastTE" class="samples-margin">TopEnd</ui5-button>
+		<ui5-toast id="wcToastTE" placement="TopEnd">TopEnd</ui5-toast>
+
+		<!--MiddleStart-->
+		<ui5-button id="wcBtnShowToastMS" class="samples-margin">MiddleStart</ui5-button>
+		<ui5-toast id="wcToastMS" placement="MiddleStart">MiddleStart</ui5-toast>
+
+		<!--MiddleCenter-->
+		<ui5-button id="wcBtnShowToastMC" class="samples-margin">MiddleCenter</ui5-button>
+		<ui5-toast id="wcToastMC" placement="MiddleCenter">MiddleCenter</ui5-toast>
+
+		<!--MiddleEnd-->
+		<ui5-button id="wcBtnShowToastME" class="samples-margin">MiddleEnd</ui5-button>
+		<ui5-toast id="wcToastME" placement="MiddleEnd">MiddleEnd</ui5-toast>
+
+		<!--BottomStart-->
+		<ui5-button id="wcBtnShowToastBS" class="samples-margin">BottomStart</ui5-button>
+		<ui5-toast id="wcToastBS" placement="BottomStart">BottomStart</ui5-toast>
+
+		<!--BottomCenter-->
+		<ui5-button id="wcBtnShowToastBC" class="samples-margin">BottomCenter</ui5-button>
+		<ui5-toast id="wcToastBC">BottomCenter</ui5-toast>
+
+		<!--BottomEnd-->
+		<ui5-button id="wcBtnShowToastBE" class="samples-margin">BottomEnd</ui5-button>
+		<ui5-toast id="wcToastBE" placement="BottomEnd">BottomEnd</ui5-toast>
+
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+
+<!--TopStart-->
+<ui5-button id="wcBtnShowToastTS">TopStart</ui5-button>
+<ui5-toast id="wcToastTS" placement="TopStart">TopStart</ui5-toast>
+
+<!--TopCenter-->
+<ui5-button id="wcBtnShowToastTC">TopCenter</ui5-button>
+<ui5-toast id="wcToastTC" placement="TopCenter">TopCenter</ui5-toast>
+
+<!--TopEnd-->
+<ui5-button id="wcBtnShowToastTE">TopEnd</ui5-button>
+<ui5-toast id="wcToastTE" placement="TopEnd">TopEnd</ui5-toast>
+
+<!--MiddleStart-->
+<ui5-button id="wcBtnShowToastMS">MiddleStart</ui5-button>
+<ui5-toast id="wcToastMS" placement="MiddleStart">MiddleStart</ui5-toast>
+
+<!--MiddleCenter-->
+<ui5-button id="wcBtnShowToastMC">MiddleCenter</ui5-button>
+<ui5-toast id="wcToastMC" placement="MiddleCenter">MiddleCenter</ui5-toast>
+
+<!--MiddleEnd-->
+<ui5-button id="wcBtnShowToastME">MiddleEnd</ui5-button>
+<ui5-toast id="wcToastME" placement="MiddleEnd">MiddleEnd</ui5-toast>
+
+<!--BottomStart-->
+<ui5-button id="wcBtnShowToastBS">BottomStart</ui5-button>
+<ui5-toast id="wcToastBS" placement="BottomStart">BottomStart</ui5-toast>
+
+<!--BottomCenter-->
+<ui5-button id="wcBtnShowToastBC">BottomCenter</ui5-button>
+<ui5-toast id="wcToastBC">BottomCenter</ui5-toast>
+
+<!--BottomEnd-->
+<ui5-button id="wcBtnShowToastBE">BottomEnd</ui5-button>
+<ui5-toast id="wcToastBE" placement="BottomEnd">BottomEnd</ui5-toast>
+
+<script>
+
+	// Attaching click listener to the Button which shows the TopStart Toast
+	wcBtnShowToastTS.addEventListener('click', function () {
```

Comment:
From the previous samples the user already know how to bind a click to ui5-button and show a toast, this is too much code for playground sample.

Either remove all the wcBtnShowToastTC.addEventListener statements, or just leave the short variation:
```js
document.querySelectorAll("ui5-button").forEach(function(button){
		button.addEventListener('click', function () {
			document.querySelector("#" + button.id.replace("BtnShow", "")).show();
		});
	});
```